documentation/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -152,7 +154,7 @@ \subsection{Electric Shock}
 \subsubsection*{Description}
 The system will require electrical components in order to activate motors. This will also include power balancing and transfer to the necessary components. This means some wiring will be needed and thus some inherent risk of shock to the user via such wiring or the sensors or actuators themselves. Only when the wires carry current would this be a risk.
 \subsubsection*{Mitigation Plan}
-All wiring will be enclosed to the best of our ability. However, due to the nature of the system requiring motion it can be difficult to completely enclose the system. Proper warnings and notices will be used to warn the user of electrical current. The system should also have a fail-safe which cuts power if it short circuits. \\ \\
+All wiring will be enclosed to the best of our ability. However, due to the nature of the system requiring motion it can be difficult to completely enclose the system. Proper warnings and notices will be used to warn the user of electrical current. The system will also be fused to prevent prolonged short circuits. \\ \\
```

Comment:
I would revert this change and say 'have a fail-safe via a fuse, which cuts power...'

Reply:
Not huge on the term "cut the power", can we use another phrasing?

Reply:
Uhhh... maybe "cease power input"?

Reply:
Lets get all the changes in asap though :)